pkg/getEc2Info.go

```diff
@@ -0,0 +1,20 @@
+package pkg
+
+import
+(	"context"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+)
+
+
+type EC2DesAPI interface {
+	DescribeInstances(ctx context.Context,
+		params *ec2.DescribeInstancesInput,
+		optFns ...func(*ec2.Options)) (*ec2.DescribeInstancesOutput, error)
+}
+
+// Returns info about ec2 instance
+func GetEc2Info(ctx context.Context, input *ec2.DescribeInstancesInput, api EC2DesAPI) (*ec2.DescribeInstancesOutput, error) {
```

Comment:
return only useful information, maybe create `EC2Instance` struct for the same which can be reused if needed

Reply:
Something like this?

``var ec2Instance [] EC2Instance``

```
type EC2Instance struct {

	InstanceName 		string
	InstanceID	 		string
	InstanceType 		string
	InstancePrivateIP 	string

}
```
		}
			ec2Instance = append(ec2Instance, EC2Instance{
				InstanceName: 		instanceName,
				InstanceID:			aws.ToString(i.InstanceId) ,
				InstanceType: 		*aws.String(string(*&i.InstanceType)),
				InstancePrivateIP: 	aws.ToString(*&i.PrivateIpAddress),
			})

Reply:
yeah that would do